spec/std/yaml/serialization_spec.cr

```diff
@@ -305,7 +305,7 @@ describe "YAML serialization" do
     end
 
     it "does for utc time with milliseconds" do
-      time = Time.new(2010, 11, 12, 1, 2, 3, 456, kind: Time::Kind::Utc)
+      time = Time.new(2010, 11, 12, 1, 2, 3, 456_000_000, kind: Time::Kind::Utc)
```

Comment:
Formatters and parsers are broken AFAIS, and I have proposed #5070. I don't know why you code pass the specs :)

Reply:
They are not broken. They just lack precision, but that's expected because at that time `Time` didn't have that precision... So #5070 is a feature, not a bug fix.